tests/schema/test_logging.py

```diff
@@ -1,8 +1,13 @@
+from unittest import skipIf, skipUnless
+
 from django.db import connection
 from django.test import TestCase
 
 
 class SchemaLoggerTests(TestCase):
+    @skipIf(
+        connection.vendor == "postgresql", reason="postgresql will merge on the client"
```

Comment:
Use a database feature flag rather than a vendor check since this applies to CockroachDB too. Rather than duplicating this as two tests, could you do something like:
```python
if connection.features.whatever:
    sql = "SELECT * FROM foo WHERE id in (42, 1337)"
    params = None
```
before the three assertions?